app/models/host/managed.rb

```diff
@@ -674,11 +674,12 @@ def root_pass
 
   def clone
     # do not copy system specific attributes
-    host = self.deep_clone(:include => [:host_config_groups, :host_classes, :host_parameters],
+    host = self.deep_clone(:include => [:host_config_groups, :host_classes, :host_parameters, :lookup_values],
                            :except  => [:name, :mac, :ip, :uuid, :certname, :last_report])
     self.interfaces.each do |nic|
       host.interfaces << nic.clone
     end
+    host.config_groups = config_groups
```

Comment:
I managed to fix this by improving the relation definitions in ConfigGroup - it needed both host/hostgroup listed as the relationship through HostConfigGroup was polymorphic for either.